File: fixtures/python_output/windows_get_with_escaped_values.py
import requests

data = {
  'a': 'b',
  'c': 'd',
  'e': 'f',
  'h': 'i',
  'j': 'k',
  'l': 'm'
}

response = requests.post('http://www.url.com/page', data=data)

File: fixtures/php_output/windows_get_with_escaped_values.php
<?php
include('vendor/rmccue/requests/library/Requests.php');
Requests::register_autoloader();
$headers = array();
$data = array(
    'a' => 'b',
    'c' => 'd',
    'e' => 'f',
    'h' => 'i',
    'j' => 'k',
    'l' => 'm'
);
$response = Requests::post('http://www.url.com/page', $headers, $data);